web/src/main/java/org/wso2/testgrid/web/api/ProductService.java

```diff
@@ -203,7 +203,7 @@ public Response isProductReportExist(
                 String bucketKey = Paths.get(Constants.AWS_BUCKET_ARTIFACT_DIR, productName, fileName).toString();
                 ArtifactReadable artifactReadable = new AWSArtifactReader(ConfigurationContext.
                         getProperty(ConfigurationContext.ConfigurationProperties.AWS_REGION_NAME),
-                        Constants.AWS_BUCKET_NAME);
+                        ConfigurationContext.getProperty(ConfigurationContext.ConfigurationProperties.AWS_BUCKET_NAME));
                 if (artifactReadable.isExistArtifact(bucketKey)) {
```

Comment:
We should rename the method "isExistArtifact" to something like "isArtifactExist" or "isExist". WDYT?